src/collections/Table/Table.js

```diff
@@ -82,6 +82,7 @@ export default class Table extends Component {
     const classes = classNames(
       'sd-table',
       'ui',
+      {sortable: !!this.props.onSortChange},
```

Comment:
If the sort handler is present, add the class so we get the proper formatting:

http://semantic-ui.com/collections/table.html#sortable